.github/workflows/ci.yml

```diff
@@ -68,6 +70,6 @@ jobs:
       run: ls && cd vcpkg && ls
       shell: bash
     - name: Prepare for build
-      run:  cd build && cmake .. -DCMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/rpc/vcpkg/scripts/buildsystems/vcpkg.cmake
+      run:  cd build && cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_TOOLCHAIN_FILE=${{ runner.workspace }}/rpc/vcpkg/scripts/buildsystems/vcpkg.cmake
```

Comment:
Is Release still in progress?

Reply:
this is for the build for us -- not the customer, no artifacts are uploaded.

Reply:
Cool, thanks!